src/Components/Home.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { getHeros, getHerosSearch } from '../Redux/HomeReducer/HomeReducer';
+import './Home.css';
+import Character from './Character';
+import Search from './Search';
+import marvelImage from '../Images/marvelImage.jpg';
+
+const Home = () => {
+  const [query, setQuery] = useState('');
+  const heros = useSelector((state) => state.heros);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (query === '') {
+      dispatch(getHeros());
+    } else {
+      dispatch(getHerosSearch(query));
+    }
+  }, [query]);
+
+  return (
+    <div className="characters-section">
+      <div className="marvel-characters-title">
+        <img className="marvel-characters-image" src={marvelImage} alt="marvel-heros" />
+        <div className="marvel-characters-text">
+          <p className="characters-text">CHARACTERS</p>
+          <p className="characters-counter">(1562)</p>
+        </div>
+      </div>
+      <div className="testing">
+        <div className="marvel-characters">
+          <div className="character-title">
+            <Search search={(q) => setQuery(q)} />
+          </div>
+          {heros.length ? heros.map((hero) => <Link state={hero} className="characters-link" key={`heroid-${hero.id}`} to="Comics"><Character newhero={hero} /></Link>) : <div className="loading">Loading...</div>}
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] while your design look stunning and awesome you should stick to the design guidelines as part of the requirements , including : 
1. Typography: font face, size, and weight
   * Layout: composition and space between elements.
1. You should select one main colour for the app and be consistent (it does **not** need to be the same colour as in the design guidelines).

Your Design           | Guidelines 
:-------------------------:|:-------------------------:
![image](https://github.com/AlyAtalla/react-capstone/assets/89282221/64cfec23-777f-4dbf-9640-351f8413c8da) | ![image](https://github.com/AlyAtalla/react-capstone/assets/89282221/15ba7c2e-3264-4ca2-a177-3584b4a9076e)